packages/@atjson/offset-annotations/src/utils/social-urls.ts

```diff
@@ -124,6 +125,25 @@ function normalizeInstagramReelURL(url: IUrl) {
   };
 }
 
+// Threads
+// - www.threads.net/:handle/post/:id
+function isThreadsURL(url: IUrl) {
+  return (
+    url.host === "threads.net" &&
+    !!url.pathname.match(/^\/[^\/]+\/[^\/]+\/[^\/]+/)
```

Comment:
I think this regex is testing that there are at least 3 path parts and that they are all non-empty, is that right? If you want to use a regex here, can you define it as a global const and give it a descriptive name?